src/ajax.js

```diff
@@ -94,6 +94,7 @@ export function ajaxBuilder(timeout = 3000, {request, done} = {}) {
       }
     } catch (error) {
       utils.logError('xhr construction', error);
+      typeof callback === 'object' && callback !== null && callback.error(error);
```

Comment:
hi @jimdigriz, we should use utils.inFn(callback.error) before to call it.

Reply:
It replicates verbatim the logic further up the function determining if the callback object is valid. If you want me to make that change, should earlier up also be the same?

My view, checks are not really necessary here as the only callers are other modules in the project and if the caller wishes to pass in garbage it should explode by design.

Reply:
Ok, let's move forward on this way.